java/mtsj/core/src/main/java/io/oasp/application/mtsj/bookingmanagement/dataaccess/api/BookingEntity.java

```diff
@@ -58,7 +57,7 @@
 
   private UserEntity user;
 
-  @LazyCollection(LazyCollectionOption.FALSE)
+  @Basic(fetch = FetchType.LAZY)
```

Comment:
We sometimes have annotations on the field itself and somestimes on the getters.
For now we simply specified all annotations on the getters. So we should do it also here consistently.
In this special case you could even integrate the fetch specification into the existing oneToMany annotation.